Cypress/cypress/integration/Parallel_Tests/Create_Workflow/recurringWorkflow.spec.js

```diff
@@ -128,7 +128,6 @@ describe("Testing the workflow schedule on a recurring basis with a target appli
 			.find("[data-cy=statsButton]")
 			.click();
 		cy.validateWorkflowInfo(workflowName, workflowNamespace, workflowSubject, agent, "Cron workflow", "Cron workflow");
-		cy.validateStatsChart();

```

Comment:
Why are we removing this?

Reply:
I have to replace it with new `cy.validateWorkflowStatsGraph()`. I have replaced it in the previous PR. I am undoing it to avoid any conflict.